borrowd_items/apps.py

```diff
@@ -4,3 +4,10 @@
 class BorrowdItemsConfig(AppConfig):
     default_auto_field = "django.db.models.BigAutoField"
     name = "borrowd_items"
+
+    def ready(self) -> None:
+        # Unfortunately this unusued import is the recommended
+        # approach when using the `@receiver` decorator; see
+        # section "Where should this code live?" in the docs:
+        # https://docs.djangoproject.com/en/5.1/topics/signals/
+        import borrowd_items.signals  # noqa
```

Comment:
I agree, this pattern is 🤢. Imports during initialization get pretty gnarly in django!